client/virtualfund_integration_test.go

```diff
@@ -95,9 +95,29 @@ func TestVirtualFundIntegration(t *testing.T) {
 		if gotFromJ != id {
 			t.Errorf("expected completed objective with id %v, but got %v", id, gotFromJ)
 		}
+
 	}
 
 	directlyFundALedgerChannel(clientA, clientI)
 	directlyFundALedgerChannel(clientI, clientB)
 
+	outcome := outcome.Exit{outcome.SingleAssetExit{
+		Allocations: outcome.Allocations{
+			outcome.Allocation{
+				Destination: types.AddressToDestination(a),
+				Amount:      big.NewInt(5),
+			},
+			outcome.Allocation{
+				Destination: types.AddressToDestination(i),
```

Comment:
I think this should be an outcome for Bob? Should we be worried that it still works without an outcome for Bob?

Reply:
You're right! I'm pretty surprised that things worked as is, we should investigate why that is.